.github/workflows/build.yml

```diff
@@ -18,7 +18,9 @@ on:
 
 env:
   DOTNET_VERSION: ${{ '8.0.201' }}
-  ENABLE_DIAGNOSTICS: true
+  # Debug logging enabled via GitHub UI during reruns or set as repo secret.
+  # See also https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging
+  ENABLE_DIAGNOSTICS: ${{ runner.debug }}
```

Comment:
Maybe this is only available in the jobs section where the runner is used? As maybe it's per runner of a job vs. the whole workflow right? Otherwise, we should open an issue on https://github.com/actions/runner/issues as it's clearly listed in the documentation, just not with a great example...